src/components/ReactTags.tsx

```diff
@@ -107,15 +117,22 @@ const ReactTags = (props: ReactTagsProps) => {
         '[Deprecated] autofocus prop will be removed in 7.x so please migrate to autoFocus prop.'
       );
     }
-    
+
     if ((autofocus || (autoFocus && autofocus !== false)) && !readOnly) {
       resetAndFocusInput();
     }
   }, [autoFocus, autoFocus, readOnly]);
 
   useEffect(() => {
     updateSuggestions();
-  }, [query, props.suggestions]);
+  }, [query, props.suggestions, props.enableFuzzySearch]);
+
+  const fuzzySuggestions = useMemo(() => {
+    const suggestionList = props.suggestions.map((suggestion) => {
+      return suggestion.id;
+    });
+    return new Fuzzy(suggestionList);
+  }, [props.suggestions]);
```

Comment:
this should only rerun when enableFuzzySearch is true and suggestions updated

Reply:
yes , make sense added here  [feat: update fuzzySearch value memo dependency](https://github.com/react-tags/react-tags/pull/985/commits/e3a80590f041b5112cab3b6938b7e12720fa4992)

https://github.com/react-tags/react-tags/blob/e3a80590f041b5112cab3b6938b7e12720fa4992/src/components/ReactTags.tsx#L137